lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -27,6 +27,12 @@
         tags:
             description: Filter results on a specific tag
             type: list
+        mandatory_tags:
+            description: Filter result always matching all these tags
+            type: list
+        exclude_tags:
```

Comment:
version_added?

Reply:
@bcoca Should I add 2.8?

Reply:
Unless you just want to aim for 2.9? 

we are still accepting changes, but you'll need 'ship it' from maintainers and we are close to freeze.

Reply:
@bcoca done
What's needed to ship it with 2.8?

Reply:
review and approval by maintainer via a 'ship it' comment  (see bot cc'ing maintainers above, bot help also gives specifics).